src/components/TokenSelect.js

```diff
@@ -33,7 +34,7 @@ const TokenSelect = (props) => {
       <TouchableHighlight
         style={index === 0 ? styles.firstItemWrapper : null}
         onPress={() => { props.onItemPress(item); }}
-        underlayColor='rgba(227, 0, 82, 0.5)'
+        underlayColor='rgba(140, 70, 255, 0.3)'
```

Comment:
Isn't it better to include as a constant as well?

Reply:
Created a method in constants to get this color depending on the alpha value.